cmd/integration_test/build/buildspecs/cloudstack-test-eks-a-cli.yml

```diff
@@ -38,13 +38,15 @@ env:
     T_PRIVATE_REGISTRY_MIRROR_CA_CERT: "harbor-registry-data:authenticated_tinkerbell_caCert"
     T_AWS_IAM_ROLE_ARN: "aws-iam-auth-role:ec2_role_arn"
     T_CLOUDSTACK_DOMAIN: "cloudstack_ci_beta_connection:domain"
+    T_CLOUDSTACK_DOMAIN_2: "cloudstack_ci_beta_connection:domain_2"
```

Comment:
I know we are just following from other providers here. Can we rename this as `T_CLOUDSTACK_MULTILEVEL_DOMAIN` or `T_CLOUDSTACK_4LEVEL_DOMAIN` to get an idea of what this is.

Reply:
Updated